tailwind.config.js

```diff
@@ -18,6 +18,33 @@ module.exports = {
       default: {
         css: {
           color: theme('colors.gray.800'),
+          marginTop: '3em',
+          a: {
+            color: theme('colors.blue.600'),
+            textDecoration: 'underline',
+          },
+          h1: {
+            lineHeight: '1.25',
+          },
+          h2: {
+            lineHeight: '1.33',
+          },
+          h3: {
+            lineHeight: '1.4',
+          },
+        },
+      },
+      xl: {
+        css: {
+          h1: {
+            lineHeight: '1.25',
+          },
+          h2: {
+            lineHeight: '1.33',
+          },
+          h3: {
+            lineHeight: '1.4',
+          },
```

Comment:
**suggestion:** reuse line heights from theme object: `lineHeight: theme('lineHeight.relaxed')` ([see available options](https://tailwindcss.com/docs/line-height))

Reply:
Yup can swap it for the theme object

We should probably pick a consistent typeface(s) for the egghead-next site sooner rather than later so we can design around it and not worry about these changing based on the typeface. Squishy line heights are kind of a killer for readability for articles so would love to get these looking good early on.

Reply:
Swapped to the theme objects :) ec24e33

Reply:
I find prose defaults to have very relaxed line heights and read well. system fonts are fine for the foreseeable future imho.